airbyte-integrations/connectors/source-tidb/src/test/java/io/airbyte/integrations/source/tidb/TiDBSourceTests.java

```diff
@@ -11,10 +11,12 @@
 import io.airbyte.cdk.db.jdbc.JdbcUtils;
 import io.airbyte.commons.json.Jsons;
 import io.airbyte.protocol.models.v0.AirbyteConnectionStatus;
+import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 import org.testcontainers.containers.GenericContainer;
 import org.testcontainers.utility.DockerImageName;
 
+@Disabled
```

Comment:
Is this deliberate? I don't understand why this test is disabled. This comment applies elsewhere in this PR as well.